mmv1/third_party/terraform/services/dataflow/resource_dataflow_job_test.go.erb

```diff
@@ -243,12 +243,12 @@ func TestAccDataflowJob_withLabels(t *testing.T) {
 			{
 				Config: testAccDataflowJob_labels(bucket, job, key, value),
 				Check: resource.ComposeTestCheckFunc(
-					testAccDataflowJobExists(t, "google_dataflow_job.with_labels"),
-					testAccDataflowJobHasLabels(t, "google_dataflow_job.with_labels", key),
+					testAccDataflowJobExists(t, "google_dataflow_job.big_data"),
```

Comment:
Just confirming that every change below this line is `with_labels` -> `big_data`, I didn't see anything else when i read through!

Reply:
That is right. with_labels -> big_data is the only change.